VALIDATORS_FAQ.md

```diff
@@ -0,0 +1,254 @@
+
+### What is a full-node?
+
+A full-node is a program that fully validates transactions and blocks of a blockchain. It is distinct from a light-node that only processes block headers and a small subset of transactions. Running a full-node requires more resources than a light-node but is necessary in order to be a validator. In practice, running a full-node only implies running a non-compromised and up-to-date version of the software with low network latency and without downtime. 
+
+### What is a delegator?
+
+Delegators are Atom holders who cannot, or do not want to run validator operations themselves. Through the Cosmos UI, a user can delegate Atoms to a validator and obtain a part of its reward in exchange.
+
+### How to become a validator?
+
+Any participant in the network can signal that they want to become a validator by sending a “declare-candidacy” transaction, where they must fill out the following parameters:
+
+* Validator's PubKey: The validator must signal an account with which it will perform its validator duties. The private key associated with PubKey is used to sign *prevotes* and *precommits*. This way, validators can have different accounts for validating and holding liquid funds.
+* Validator's name
+* Validator's website (Optional)
+* Validator's description (Optional)
+* Initial commission rate: The commission rate of fees charged to any delegators
+* Maximum commission: The maximum commission rate which this validator candidate can charge
+* Commission change rate: The maximum daily increase of the validator candidate commission
+* Minimum self-bond amount: Minimum amount of Atoms the validator candidate need to have bonded at all time. If the validator's self-bonded stake falls below this limit, its entire staking pool will unbond.
+* Initial self-bond amount: Initial amount Atoms the validator candidate wants to self-bond
+
+Once a PubKey has declared candidacy, Atom holders can delegate atoms to it, effectively adding stake to this pool. The total stake of an address is the combination of Atoms bonded by users (called delegators) and Atoms self-bonded by the entity which designated itself.
+
+Out of all the candidates that signaled themselves, the 100 with the most stake are the ones who are designated as validators. If a validator’s total stake falls below the top 100 then that validator loses its validator privileges. Over time, the maximum number of validators will increase, according to a predefined schedule:
+
+* **Year 0:** 100
+* **Year 1:** 113
+* **Year 2:** 127
+* **Year 3:** 144
+* **Year 4:** 163
+* **Year 5:** 184
+* **Year 6:** 208
+* **Year 7:** 235
+* **Year 8:** 265
+* **Year 9:** 300
+* **Year 10:** 300
+
+### Is there a minimum amount of Atoms that must be staked to be a validator?
+
+There is no minimum. The top 100 validator candidates with the highest total stake (where total stake = self-bonded stake + delegators stake) are the validators. 
+
+### How will delegators choose their validators?
```

Comment:
We should s tart this with what it means to be a good validator at all, and emphasize delegators are responsible for uiltimately selecting the validator set and must stay active and vigilant in doing so

Reply:
Good validator is explained below. As for delegators' responsibilities, I added it in delegators section above

Reply:
How is the validator # calculated? Maybe state that the total # is subject to change, based on network load/usage, desire to maintain certain performance metrics, etc.